src/Education/XamlGenerator/IRuleHelpXamlBuilder.cs

```diff
@@ -217,14 +216,14 @@ private void ProcessElement()
 
                 case "p":
                     writer.WriteStartElement("Paragraph");
-                    writer.WriteAttributeString("xml", "space", null, "default");
```

Comment:
I've made a few changes to how whitespace is being handled, having looked at some sample generated rule help.

Previously, whitespace was being `preserved` at the top-level, and turned off (`default`) on specific child cells i.e. most whitespace (particularly line breaks) would be rendered in the output. However, that was leading to line breaks appearing in odd places, so I've flipped it round.

Now, whitespace between tags is ignored (the default). It is only preserved for specific child elements, namely those inside the `pre` <html> tag.

Reply:
Makes sense